ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClient.java

```diff
@@ -74,11 +73,12 @@ void createTopic(
   Map<String, TopicDescription> describeTopics(Collection<String> topicNames);
 
   /**
-   * [warn] synchronous call to get the response
+   * [warn] Asynchronous call to get the response
    *
-   * @param topicName topicName to describe
+   * @param topicName topicNames to describe
    */
-  DescribeConfigsResult describeConfigs(String topicName);
+  public String getTopicCleanupPolicy(String topicName);
```

Comment:
Would be better if this was an enum with three values: `delete`, `compact`, `compact+delete`. Would be better than each user of the method relying on the actual string literals.